makefile

```diff
@@ -60,13 +60,13 @@ LOCAL = $(TESTS) $(CWARNS)
 
 
 # enable Linux goodies
-MYCFLAGS= $(LOCAL) -std=c99 -DLUA_USE_LINUX -DLUA_USE_READLINE
-MYLDFLAGS= $(LOCAL) -Wl,-E
-MYLIBS= -ldl -lreadline
+MYCFLAGS= $(LOCAL) -std=c99 -DLUA_USE_LINUX -march=rv64imafdcxcheri -mabi=l64pc128d --sysroot=$(HOME)/cheri/output/rootfs-riscv64-purecap -mno-relax -g
+MYLDFLAGS= $(LOCAL) -Wl,-E --sysroot=$(HOME)/cheri/output/rootfs-riscv64-purecap -mabi=l64pc128d
+MYLIBS= -ldl
 
 
-CC= gcc
-CFLAGS= -Wall -O2 $(MYCFLAGS) -fno-stack-protector -fno-common -march=native
+CC=$(HOME)/cheri/output/sdk/bin/riscv64-unknown-freebsd13-clang
+CFLAGS= -Wall -O0 $(MYCFLAGS) -fno-stack-protector -fno-common
```

Comment:
Why `-O0`? For easier debugging?

Reply:
Yes, I think it might be fine to leave it like that for now, but not sure?

Reply:
Do things run OK with `-O2`? In which case I'd be inclined to leave things at `-O2`, otherwise we'll eventually forget we change it to `-O0` and then we'll benchmark the slow version by accident!